app/views/notes/_comment.html.erb

```diff
@@ -64,7 +64,7 @@
   </script>
   
   <div id="c<%= comment.cid %>show">
-    <p><%= raw sanitize(auto_link(RDiscount.new(comment.body).to_html)) %></p>
+    <p><%= raw sanitize (RDiscount.new(title_suggestion(comment.body, comment.drupal_user.name, comment.nid, comment.id)).to_html), attributes: %w(class style href data-method) %></p>
```

Comment:
This isi getting quite long; any way to simplify via formatting?

Reply:
Maybe pass the comment object to title_suggestion method would be better rather than breaking it up into body, name, etc. 'Attributes' require href and data-method for the time being perhaps. I'll try something to shorten this!!,